package.json

```diff
@@ -23,7 +23,10 @@
     "test:e2e": "BUILD_MODE=e2e-test yarn build && (cd test/app && rm -rf node_modules && yarn && yarn build) && wdio test/e2e/wdio.local.conf.js",
     "test:e2e:bs": "BUILD_MODE=e2e-test yarn build && (cd test/app && rm -rf node_modules && yarn && yarn build) && node ./scripts/bs-wrapper.js wdio test/e2e/wdio.bs.conf.js",
     "test:compat:tsc": "yarn build && (cd test/app && rm -rf node_modules && yarn && yarn compat:tsc) || yarn fail 'typescript 3.0 compatibility broken'",
-    "test:compat:ssr": "yarn build && (cd test/app && rm -rf node_modules && yarn && yarn compat:ssr) || yarn fail 'server side rendering compatibility broken'"
+    "test:compat:ssr": "yarn build && (cd test/app && rm -rf node_modules && yarn && yarn compat:ssr) || yarn fail 'server side rendering compatibility broken'",
+    "submodule:update": "git submodule update --init && git submodule update --remote",
+    "json2type:build": "cd ./node_modules/json-schema-to-typescript && rm -rf dist && npm i; npm run clean && npm run build:server; echo '>>> errors due to build inside node_modules, but it should still work fine'",
```

Comment:
I don't understand why do we need to rebuild this dependency. Could you please explain what is the problem with using the originally built files?

Reply:
It is because I am using a forked version, the files are not built when installing the package from github instead of npm

Reply:
This is because the dependency is on a git branch, not a package published on NPM, and yarn don't build the package when installing. This is not pretty, but since it is temporary while the PR is not merged upstream, I guess it's fine!

Reply:
Yes, I prefer to have this ugly rebuild than having to publish a new package or commit the dist files on the fork